testing/jormungandr-testing-utils/src/testing/node/rest/load.rs

```diff
@@ -112,7 +116,13 @@ impl RequestGenerator for RestRequestGen {
         })
     }
 
-    fn split(self) -> (Self, Option<Self>) {
+    fn split(mut self) -> (Self, Option<Self>) {
+        // Since rest queries do not modify the node state we can split as many times as we want
+        // but that may trigger a bug in rayon so we artificially limit it
```

Comment:
Is there a benefit of using rayon at all?

Reply:
Maybe @Zeegomo can anwser that. From my perspective it increased load tool transaction volume greatly, but don't know if that was by using framework or fixing my code issues